kraftwerk-core/src/main/java/fr/insee/kraftwerk/core/utils/SqlUtils.java

```diff
@@ -0,0 +1,294 @@
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+@Slf4j
+public class SqlUtils {
+
+    private SqlUtils() {
+        throw new IllegalStateException("Utility class");
+    }
+
+    /**
+     * Convert vtl bindings to SQL DuckDB tables for further export
+     * @param vtlBindings vtl bindings to send into database
+     * @param statement statement associated to database
+     */
+    public static void convertVtlBindingsIntoSqlDatabase(VtlBindings vtlBindings, Statement statement) {
+        try {
+            createOutputSQLTables(statement, vtlBindings);
+        } catch (SQLException e) {
+            log.error("SQL Error during VTL bindings conversion :\n{}",e.toString());
+        }
+    }
+
+
+    /**
+     * send CREATE TABLE queries into DB
+     *
+     * @param statement   DuckDB connection
+     * @param vtlBindings datasets to convert
+     * @throws SQLException if sql error
+     */
+    private static void createOutputSQLTables(Statement statement, VtlBindings vtlBindings) throws SQLException {
+        for (String datasetName : vtlBindings.getDatasetNames()) {
+            //Variables types map
+            Map<String, VariableType> sqlSchema = extractSqlSchema(vtlBindings.getDataset(datasetName).getDataStructure());
+            //Skip if no variable
+            if (sqlSchema.isEmpty()) {
+                log.warn("Empty schema for dataset {}", datasetName);
+            } else {
+                //Column order map to use in INSERT VALUES statement
+                List<String> schemaOrder = extractColumnsOrder(vtlBindings.getDataset(datasetName));
+
+                //Skip CREATE if table already exists (ex: file-by-file)
+                List<String> tableNames = getTableNames(statement);
+                if (!tableNames.contains(datasetName)) {
+                    //CREATE query building
+                    StringBuilder createTableQuery = new StringBuilder(String.format("CREATE TABLE '%s' (", datasetName));
+
+                    for (String columnName : schemaOrder) {
+                        createTableQuery.append("\"").append(columnName).append("\"").append(" ").append(sqlSchema.get(columnName).getSqlType());
+                        createTableQuery.append(", ");
+                    }
+
+                    //Remove last delimiter and replace by ")"
+                    createTableQuery.delete(createTableQuery.length() - 2, createTableQuery.length());
+                    createTableQuery.append(")");
+
+                    //Execute query
+                    log.debug("SQL Query : {}", createTableQuery);
+                    statement.execute(createTableQuery.toString());
+                }
+                insertDataIntoTable(statement.getConnection(), datasetName, vtlBindings.getDataset(datasetName), schemaOrder);
```

Comment:
I prefer to call this method in convertVtlBindingsIntoSqlDatabase. 
First create tables
Next insert data